src/components/ViewDao/ViewDaoContainer.tsx

```diff
@@ -184,8 +184,8 @@ class ViewDaoContainer extends React.Component<IProps, IState> {
 
     this.executeProposalEventWatcher = votingMachineInstance.ExecuteProposal({}, { fromBlock: "latest" });
```

Comment:
I think you should use `GP.ExecutedProposals`, not `GP.ExecuteProposal`.  It will give you the executionState and a nice object entity for all of the event arguments. You will save yourself a bunch of lines of code and possibly fewer changes required overall for this PR.

Reply:
https://github.com/daostack/alchemy/pull/122/commits/4744d57fded5c62ebdbe4de258a0eafb39baf33a